src/main/java/org/hmhb/program/Program.java

```diff
@@ -65,7 +65,7 @@
                     referencedColumnName = "id"
             )
     )
-    private List<County> counties;
+    private List<County> countiesServed;
```

Comment:
The client was passing in countiesServed, but the server was expecting counties.  I fixed the bug and retested.